services/apps/portal/src/providers/typeorm/migrations/1674502121879-add_owners.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addOwners1674502121879 implements MigrationInterface {
+  name = 'addOwners1674502121879';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "portal"."application_owner_type" ("audit_deleted_date_at" TIMESTAMP WITH TIME ZONE, "audit_created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "audit_updated_at" TIMESTAMP WITH TIME ZONE DEFAULT now(), "audit_created_by" character varying NOT NULL, "audit_updated_by" character varying, "label" character varying NOT NULL, "code" text NOT NULL, "description" text NOT NULL, CONSTRAINT "UQ_1226285ea7299ecd8980e56ace8" UNIQUE ("description"), CONSTRAINT "PK_05181ec6491ee0aa527bd55c714" PRIMARY KEY ("code"))`,
+    );
+    await queryRunner.query(
+      `CREATE TABLE "portal"."application_owner" ("audit_deleted_date_at" TIMESTAMP WITH TIME ZONE, "audit_created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "audit_updated_at" TIMESTAMP WITH TIME ZONE DEFAULT now(), "audit_created_by" character varying NOT NULL, "audit_updated_by" character varying, "uuid" uuid NOT NULL DEFAULT gen_random_uuid(), "first_name" character varying, "last_name" character varying, "organization_name" character varying, "phone_number" character varying, "email" character varying, "type_code" text NOT NULL, CONSTRAINT "PK_9b2fecd11351ea787e05900672d" PRIMARY KEY ("uuid"))`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "portal"."application_parcel" ADD "owner_uuid" uuid`,
```

Comment:
do I get it right that this is an assigned owner of a parcel? I'm asking since I can see application_parcel_owners_application_owner table that also holds similar relationship

Reply:
oh good catch, maybe this was created in an earlier version and I should delete it. I did originally have a single owner per parcel and I think this might be from that.

Reply:
Cleaned up